test/test.js

```diff
@@ -162,6 +180,10 @@ describe('chai-immutable (' + typeEnv + ')', function () {
         fail(function () { expect(deepMap).to.not.eqls(sameDeepMap); });
         fail(function () { expect(deepMap).to.not.deep.equal(sameDeepMap); });
       });
+
+      it('should work if using different copies of Immtuable', function () {
+        expect(clonedImmutableList).to.equal(clonedImmutable.List.of(1, 2, 3));
```

Comment:
Shouldn't this test make sure that an identical list from 2 different versions of Immutable.js are the same?
If so then maybe it should be:

```js
expect(clonedImmutableList).to.equal(list3);
```

No?